ladybug_display_schema/graphic.py

```diff
@@ -0,0 +1,294 @@
+    energy_flux = 'EnergyFlux'
+    energy_intensity = 'EnergyIntensity'
+    enthalpy = 'Enthalpy'
+    extraterrestrial_direct_normal_radiation = 'ExtraterrestrialDirectNormalRadiation'
+    extraterrestrial_horizontal_radiation = 'ExtraterrestrialHorizontalRadiation'
+    fraction = 'Fraction'
+    global_horizontal_illuminance = 'GlobalHorizontalIlluminance'
+    global_horizontal_irradiance = 'GlobalHorizontalIrradiance'
+    global_horizontal_radiation = 'GlobalHorizontalRadiation'
+    ground_temperature = 'GroundTemperature'
+    heating_degree_time = 'HeatingDegreeTime'
+    horizontal_infrared_radiation_intensity = 'HorizontalInfraredRadiationIntensity'
+    humidity_ratio = 'HumidityRatio'
+    illuminance = 'Illuminance'
+    irradiance = 'Irradiance'
+    liquid_precipitation_depth = 'LiquidPrecipitationDepth'
+    liquid_precipitation_quantity = 'LiquidPrecipitationQuantity'
+    luminance = 'Luminance'
+    mass = 'Mass'
+    mass_flow_rate = 'MassFlowRate'
+    mean_radiant_temperature = 'MeanRadiantTemperature'
+    metabolic_rate = 'MetabolicRate'
+    opaque_sky_cover = 'OpaqueSkyCover'
+    operative_temperature = 'OperativeTemperature'
+    operative_temperature_delta = 'OperativeTemperatureDelta'
+    percentage_people_dissatisfied = 'PercentagePeopleDissatisfied'
+    power = 'Power'
+    precipitable_water = 'PrecipitableWater'
+    predicted_mean_vote = 'PredictedMeanVote'
+    pressure = 'Pressure'
+    prevailing_outdoor_temperature = 'PrevailingOutdoorTemperature'
+    r_value = 'RValue'
+    radiant_coefficient = 'RadiantCoefficient'
+    radiant_temperature = 'RadiantTemperature'
+    radiant_temperature_delta = 'RadiantTemperatureDelta'
+    radiation = 'Radiation'
+    relative_humidity = 'RelativeHumidity'
+    sky_temperature = 'SkyTemperature'
+    snow_depth = 'SnowDepth'
+    specific_energy = 'SpecificEnergy'
+    speed = 'Speed'
+    standard_effective_temperature = 'StandardEffectiveTemperature'
+    temperature = 'Temperature'
+    temperature_delta = 'TemperatureDelta'
+    temperature_time = 'TemperatureTime'
+    thermal_comfort = 'ThermalComfort'
+    thermal_condition = 'ThermalCondition'
+    thermal_condition_eleven_point = 'ThermalConditionElevenPoint'
+    thermal_condition_five_point = 'ThermalConditionFivePoint'
+    thermal_condition_nine_point = 'ThermalConditionNinePoint'
+    thermal_condition_seven_point = 'ThermalConditionSevenPoint'
+    time = 'Time'
+    total_sky_cover = 'TotalSkyCover'
+    utci_category = 'UTCICategory'
+    u_value = 'UValue'
+    universal_thermal_climate_index = 'UniversalThermalClimateIndex'
+    visibility = 'Visibility'
+    voltage = 'Voltage'
+    volume = 'Volume'
+    volume_flow_rate = 'VolumeFlowRate'
+    volume_flow_rate_intensity = 'VolumeFlowRateIntensity'
+    wet_bulb_temperature = 'WetBulbTemperature'
+    wind_direction = 'WindDirection'
+    wind_speed = 'WindSpeed'
+    zenith_luminance = 'ZenithLuminance'
+
+
+class DataTypeBase(BaseModel):
```

Comment:
Why not calling this DataType? I don't see any subclasses of this base class.

Reply:
This is a good point and I guess this was just a hold over from when we first put together the DataType module some ~3 years ago. I think I can change it from DataTypeBase to DataType and everything should still work.